### PR TITLE
feat(athena-driver): add environment variable to define the Athena workgroup

### PR DESCRIPTION
**Pull request by ndrluis**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

Add configuration to define the name of the workgroup in which the query is being started

### CODE REVIEW

**Comment by rpaik** [comment]
@hassankhan/@ovr can you help with the review?

File: docs/content/Configuration/Databases/AWS-Athena.md [hunk 1]
```diff
@@ -32,6 +33,7 @@ CUBEJS_AWS_S3_OUTPUT_LOCATION=s3://my-athena-output-bucket
 | `CUBEJS_AWS_SECRET`             | The AWS Secret Access Key to use for database connections         | A valid AWS Secret Access Key          |    ✅    |
 | `CUBEJS_AWS_REGION`             | The AWS region of the Cube.js deployment                          | [A valid AWS region][aws-docs-regions] |    ✅    |
 | `CUBEJS_AWS_S3_OUTPUT_LOCATION` | The S3 path to store query results made by the Cube.js deployment | A valid S3 path                        |    ❌    |
+| `CUBEJS_AWS_ATHENA_WORKGROUP`   | The name of the workgroup in which the query is being started     | A valid Athena Workgroup               |    ❌    |
```
Comment (hassankhan) [inline docs/content/Configuration/Databases/AWS-Athena.md:36]: Might be good to link "A valid Athena Workgroup" out to https://docs.aws.amazon.com/athena/latest/ug/workgroups-benefits.html

File: docs/content/Configuration/Environment-Variables-Reference.md [hunk 2]
```diff
@@ -83,6 +83,7 @@ databases [in this guide][link-connecting-to-db].
...
 | `CUBEJS_AWS_S3_OUTPUT_LOCATION`                 | AWS Athena, JDBC     | The S3 path to store query results made by the Cube.js deployment                                                                                                                                           | A valid S3 path                                                  |
 | `CUBEJS_AWS_SECRET`                             | AWS Athena, JDBC     | The AWS Secret Access Key to use for database connections                                                                                                                                                   | A valid AWS Secret Access Key                                    |
+| `CUBEJS_AWS_ATHENA_WORKGROUP`                   | AWS Athena           | The name of the workgroup in which the query is being started                                                                                                                                               | A valid Athena Workgroup                                         |
```
Comment (hassankhan) [inline docs/content/Configuration/Environment-Variables-Reference.md:86]: Same for the text here

**Review by hassankhan — COMMENTED** [review]
Minor nit; thanks for your contribution @ndrluis

**Review by hassankhan — APPROVED** [review]

**Comment by ovr** [comment]
Thank you @ndrluis for your contribution! 👍 🍰 🥮

**Pull request merged**